int16 senderID
int16 receiverID
string variableName
int16 msgID

